src/core/backbone_router/bbr_local.cpp

```diff
@@ -130,8 +130,10 @@ Error Local::SetConfig(const BackboneRouterConfig &aConfig)
     Error error  = kErrorNone;
     bool  update = false;
 
+#if !OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE
```

Comment:
We also have other logics that make sure the effective MLR timeout is always larger than `Mle::kMlrTimeoutMin`. 

Do we need to change these places too?

- Even if an small MLR timeout is configured on Leader, OT devices will use an effective MLR timeout within the valid range (in UpdateBackboneRouterPrimary). 
- In HandleMulticastListenerRegistration, MLR timeout is normalized to be in within the valid range.

Reply:
The test case is quite simple, only requires that the Leader/PBBR configures an MLR timeout of 75 seconds and verifies that the TD/DUT uses the minimum of 300 seconds anyway. With the current change the test is passing.

Reply:
Make sense to me.